.travis.yml

```diff
@@ -1,31 +1,38 @@
 language: python
+
 python:
-- 2.7
-- 3.5
+  - 2.7
+  - 3.5
 env:
-- TOXENV=django18-celery3
-- TOXENV=django19-celery3
-- TOXENV=django110-celery3
+  - TOXENV=django111
 matrix:
   include:
+  - python: 3.5
+    env: django20
+  - python: 3.5
+    env: django21
+  - python: 3.5
+    env: django22
```

Comment:
These should be formatted like `TOXENV=django22`; written as is, some of the Travis workers are each running all of the tox environments.  Example: https://travis-ci.org/edx/edx-celeryutils/jobs/619638003